tests/test_smoke.py

```diff
@@ -309,6 +309,22 @@ def test_n_node_job_queue():
     run_one_test(test)
 
 
+def test_large_job_queue():
+    name = _get_cluster_name()
+    test = Test(
+        'job_queue_large',
```

Comment:
nit: We can start making the test names and func names the same. Found it easier to find corresponding tests from logs.

Reply:
Good point. Fixed. Thanks!